client/src/App.jsx

```diff
@@ -1,10 +1,38 @@
-import './App.css'
+import { useState, useEffect } from 'react';
+import { apiUrl } from './utils';
+import './App.css';
+import ViewBlogsPreview from './components/ViewBlogsPreview';
+import CreateBlogPost from './components/CreateBlogPost';
+import ViewIndividualBlogModal from './components/ViewIndividualBlogModal';
+
 
 function App() {
 
+  const [note, setNote] = useState('');
+
+  const fetchNote = async () => {
+    const response = await fetch(`${apiUrl}`);
+    const data = await response.json();
+    setNote(data);
+  }
+
+  useEffect(() => {
+    fetchNote();
+  }, []);
+
   return (
     <>
-    <h1>Hello Whitney-Rene</h1>
+
+    <h4>Whitney-Rene's Spain Blog</h4>
+    
+    <ViewBlogsPreview />
+
+    <ViewIndividualBlogModal />
+
+    <CreateBlogPost />
+
+    <p>{note}</p>
```

Comment:
needs to be indented once as it's within the React.Fragment

Reply:
Perfect!  I needed that reminder.